django_project_base/account/authentication.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import authentication
+
+
+class SessionAuthentication(authentication.TokenAuthentication):
```

Comment:
Tole mi je suspect: zakaj ga sploh definiraš in če ga že definiraš, zakaj derivaš iz TokenAuthentication?